common/djangoapps/third_party_auth/tests/utils.py

```diff
@@ -80,7 +85,7 @@ def _setup_provider_response_with_body(self, status, body):
 class ThirdPartyOAuthTestMixinFacebook(object):
     """Tests oauth with the Facebook backend"""
     BACKEND = "facebook"
-    USER_URL = FacebookOAuth2.USER_DATA_URL
+    USER_URL = FacebookOAuth2.USER_DATA_URL.format(version=FACEBOOK_API_VERSION)
```

Comment:
Decided to take this route because the settings function says to only keep provider-independent settings in it. Either way if we upgrade to a new PSA version in the future we will presumably play along with whatever FB API version they're using. Or am I wrong to assume that? If we would, then this'll save some potential future maintainer headaches. Might have to flesh this out more carefully though because the FB API is used in one more place.